sql/moz-fx-data-shared-prod/contextual_services_derived/request_payload_suggest_v1/query.sql

```diff
@@ -0,0 +1,45 @@
+WITH parsed AS (
+  SELECT
+    submission_timestamp,
+    udf_js.parse_sponsored_interaction(udf_js.extract_string_from_bytes(payload)) AS si
+  FROM
+    `moz-fx-data-shared-prod.payload_bytes_error.contextual_services`
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND error_type = 'SendRequest'
+),
+extracted AS (
+  SELECT
+    si.formFactor AS form_factor,
+    si.interactionType AS send_event,
+    si.flaggedFraud,
+    DATE(submission_timestamp) AS submission_date,
+    CASE
+      WHEN si.source = 'topsites'
```

Comment:
since we're splitting these. `source` will never be `topsites` so this will always be a null value. I would either remove the column from this table or do something like `null as begin_timestamp` with a comment for why it's set to null